code/weapon/weapons.cpp

```diff
@@ -7534,6 +7512,33 @@ void validate_SSM_entries()
 	}
 }
 
+int weapon_get_random_usable_weapon()
```

Comment:
Maybe call this `weapon_get_random_player_usable_weapon` to make it clearer what exactly this returns.

Reply:
Ok.